rust/phoenix-sdk/src/sdk_client.rs

```diff
@@ -229,13 +183,26 @@ impl SDKClient {
     }
 
     pub async fn get_market_state(&self) -> MarketState {
-        let mut market_account_data = (self.client.get_account_data(&self.active_market_key))
-            .await
-            .unwrap();
-        let (header_bytes, bytes) = market_account_data.split_at_mut(size_of::<MarketHeader>());
-        let header = MarketHeader::try_from_slice(header_bytes).unwrap();
-        let market = load_with_dispatch_mut(&header.market_size_params, bytes)
-            .unwrap()
+        let market_account_data =
+            match (self.client.get_account_data(&self.active_market_key)).await {
+                Ok(data) => data,
+                Err(_) => {
+                    return MarketState {
```

Comment:
why do we return this instead of an explicit error?

Reply:
it's been a while since this PR so I don't fully remember the context, but I think this function was causing bots to crash if there were issues with the underlying RPC connection